shinydashboard/lantern/functions/endpoints.R

```diff
@@ -404,7 +404,7 @@ get_endpoint_security_counts <- function(db_connection) {
     "Total Indexed Endpoints", as.integer(isolate(app_data$fhir_endpoint_totals()$all_endpoints)),
     "Endpoints with successful response (HTTP 200)", as.integer(isolate(app_data$response_tally()$http_200)),
     "Endpoints with unsuccessful response", as.integer(isolate(app_data$response_tally()$http_non200)),
-    "Endpoints without valid capability statement", as.integer(get_no_cap_statement_count(db_connection)),
+    "Endpoints without valid capability statement / conformance statement", as.integer(get_no_cap_statement_count(db_connection)),
```

Comment:
Based on Jeff's comment, it seems like anywhere where we had "capability statement" it should be CapabilityStatement

Reply:
Also "conformance statement" should just be "Conformance"

Reply:
It also seems like here and basically everywhere else we should be referring to them as "resources", so this should read "Endpoints without valid CapabilityStatement/Conformance Resource"

Reply:
So I'd apply this here and all of the changes below!

Reply:
Yeah this makes much more sense, thank you for clarifying! Just made all these changes throughout all the files!